### ISSUE TITLE
Limit meetings: Timeline limitation designed

### ISSUE THREAD

**Issue by Dschoordsch** [ParabolInc/parabol#7358]
The meeting history shall be limited to 30/90 days on the timelline.

## Acceptance criteria
* [ ] design for the timeline with limits and upgrade message
* Proposal:
  * [ ]  showing at least 2 messages: one after 30 days old meetings and one after 90 days
  * [ ] timeline shows meetings <= 30 days old as it currently does
  * [ ] after the first 30 days meetings a message is shown stating that the history is limited and should be upgraded
  * [ ] more history is shown till 90 days but unaccessible, links should go to upgrade
  * [ ] after 90 days of meetings a message is shown with sth. like "upgrade to see more meetings"

**Comment by Dschoordsch** [comment]
Let's only do one hard limit for now to get something over the finish line.
@acressall what do you think about this end card?
![Screenshot from 2022-11-15 17-48-16](https://user-images.githubusercontent.com/7331043/201978587-b5e8fac3-8a68-49a4-a86c-5b5580dd55a5.png)

**Comment by acressall** [comment]
I like this a lot @Dschoordsch ! The icon just feels a tiny bit too large. [Here's a similar card](https://www.figma.com/file/IKMfMmMmOxDUWrHfsVgxU740/Timeline-Discovery?node-id=171%3A21521&t=7eKCUfRIPuzoLz3g-4) with a large icon that we can use as a baseline. 

@avivapinchas any feedback on copy?

**Comment by avivapinchas** [comment]
I'm into this! 

Copy suggestion: 
> **Past Meetings Locked**
> Your plan includes 30-days of meeting history. Unlock more by upgrading. 
> [Unlock past meetings] 

One Q on design: Is there any way to show (either real or dummy) timeline events that are blurred out? So you can kinda see a teaser of previous meeting events, similar to what we do in the Check-in Meeting phase with the agenda, before we reach that phase?

**Comment by avivapinchas** [comment]
I like that too! Basically anything that kind of teases what they are missing out on

**Comment by Dschoordsch** [comment]
Updated with locked meetings:
![Screenshot from 2022-11-22 12-05-59](https://user-images.githubusercontent.com/7331043/203298970-19eb7747-da4c-4d8b-ab3e-cbceb4333272.png)
And here also a check-in:
![Screenshot from 2022-11-22 12-06-34](https://user-images.githubusercontent.com/7331043/203298983-00fc231e-65af-443d-91a0-56381f2d2871.png)

One thing I missed before is that a user can be in both paid and unpaid teams and then the history could contain locked and unlocked items. I would anyways show the upgrade message if any locked meetings are present.

**Comment by acressall** [comment]
This is a nice iteration! 

> One thing I missed before is that a user can be in both paid and unpaid teams and then the history could contain locked and unlocked items. I would anyways show the upgrade message if any locked meetings are present.

Should we specify in the upgrade card which org we're talking about? For both this and the modal version?

**Comment by acressall** [comment]
One other thought: maybe we make the icon the grape colour, like you did in the upgrade card? Right now I tend to skip over the icon and really have to read to see which ones are locked. A little bit of colour differentiation might help

**Comment by Dschoordsch** [comment]
How about this:
![Screenshot from 2022-11-22 21-09-02](https://user-images.githubusercontent.com/7331043/203411428-54f9bcfd-66af-4cfe-aec8-17b5f5063544.png)

@ackernaut could you also check if this design is ok?

**Comment by ackernaut** [comment]
Hey @Dschoordsch 

I looked at this briefly then reviewed #7359 first. I see now how similar they are. Some details from [this review comment](https://github.com/ParabolInc/parabol/issues/7359#issuecomment-1330924571) still apply.
- I see you are using the lock icon in both places. The continuity makes sense. An illustration may feel a bit heavy in this view (as suggested in the other comment)
- Once again it would be nice to find a way to avoid single words on a line, if possible. A break between sentences may work if it doesn’t look bad in a narrow mobile-sized viewport
- This heading ‘Past Meetings Locked’ could use a similar bump in size (if you do that to the dialog view), but I see how it matches the cards below. I think it’s okay to be larger in this CTA card.
- It’s quirky but I think I like how the purple lock icon ties the 2 together. Maybe try keeping that for a while. Otherwise I thought it kind of odd to change the meaning of this icon from meeting type to locked state of the meeting history; however the meaning of the other icon (vague meaning around meeting type) isn’t that strong. This may all get a refresh if we move toward Enrique’s designs so in the spirit of iteration, safe to try!

**Comment by Dschoordsch** [comment]
@ackernaut thanks for checking. We will track the events. The link will go to the billing page. I also added the team name here as it might be ambiguous otherwise.
![Peek 2022-11-30 18-07](https://user-images.githubusercontent.com/7331043/204899055-0275e5ed-773b-4ee0-9b4c-9260d9ab8b95.gif)

**Comment by ackernaut** [comment]
I think it’s looking good! What happens if they are blocked on multiple teams? Is the copy conditional?

**Comment by Dschoordsch** [comment]
I think this is very rare. I've opted to show the message just before the first locked meeting with the team name of that meeting, i.e. the newest locked one.
If the timeline contains 3 teams, 1 is pro, 2 and 3 are locked, then the message might only mention team 2, but meetings of 2 and 3 will appear with a lock and clicking on those will lead to the appropriate team upgrade screen. We could show it multiple times, one for each locked team if you think that's better.

**Comment by ackernaut** [comment]
Yeah, I can see how it’s rare. I think showing the CTA once is fine. That GIF does say ‘Unlock’ twice in that second sentence.

**Comment by Dschoordsch** [comment]
Thanks, will fix that.
